app/build.gradle

```diff
@@ -1,12 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-  compileSdkVersion 33
-  buildToolsVersion "30.0.3"
```

Comment:
Where did (the logic of) this line go?

Reply:
It's deprecated and no longer needed as the gradle plugin behind the scenes now knows which build tools version to use.